vulcanai2/models/basenetwork.py

```diff
@@ -304,20 +298,13 @@ def fit(self, train_loader, val_loader, epochs,
         :param val_loader: The DataLoader object containing the validation data
         :param epochs: The number of epochs
         :param retain_graph: Specifies whether retain_graph will be true when .backwards is called.
-        :param valid_interv:
+        :param valid_interv: Specifies when validation should occur. Not yet implemented.
```

Comment:
I believe @priyatharsan implemented it recently. We just need to fix how we omit printing validation scores that don't exist in-between the interval